src/configs/all.ts

```diff
@@ -117,6 +117,7 @@ export const rules = {
     "no-use-before-declare": true,
     "no-var-keyword": true,
     "no-void-expression": true,
+    "prefer-conditional": true,
```

Comment:
Ok, works. The ESTree spec calls it a [conditional expression](https://github.com/estree/estree/blob/master/es5.md#conditionalexpression) 👍